src/components/Users/UserListAndCard.tsx

```diff
@@ -134,8 +134,11 @@ export const UserStatusIndicator = ({
   className?: string;
   addPadding?: boolean;
 }) => {
-  const cur_online = isUserOnline(user);
+  const authUser = useAuthUser();
+  const isAuthUser = user.id === authUser.id;
+  const cur_online = isUserOnline(user) || isAuthUser;
```

Comment:
Can we ensure proper naming conventions being used?

ref: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variable_names

```suggestion
  const isOnline = isUserOnline(user) || isAuthUser;
```

Reply:
Thank you for pointing out @rithviknishad, I have fixed it.